treap.cabal

```diff
@@ -79,10 +79,13 @@ test-suite treap-test
                          Test.Rand.Cuts
                          Test.Rand.Query
                          Test.Rand.Update
+                         Test.Rand.Laws
 
   build-depends:       treap
                      , hspec
                      , hspec-core
+                     , hedgehog >= 0.6
+                     , hw-hspec-hedgehog
```

Comment:
Nice, I haven't heard about this library. Integrating `hedgehog` with `hspec` can make things simpler

Reply:
it's a pretty new library but I figured it was better than adding tasty just to run hedgehog :)